.github/CODEOWNERS

```diff
@@ -1,10 +1,10 @@
 # Default owners for everything
 
-* @abbyoung @esacteksab @gidjin
+* @abbyoung @esacteksab @gidjin @minhlai
 
 # Github settings
 
-.github/settings.yml @abbyoung @esacteksab @noahfirth @gidjin
+.github/settings.yml @abbyoung @esacteksab @noahfirth @gidjin @minhlai
```

Comment:
Suggestion(non-blocking): drop Noah from the list of owners of the settings file.

Reply:
Done!